src/panels/detail/components/description/DetailPanelLinkDescription.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+interface Props {}
+
+export const DetailPanelLinkDescription: React.FC<Props> = (props: Props) => {
+  return <div></div>;
+};
```

Comment:
Empty component that returns div? I mean, why bother with the component and not just return the div